src/test/java/com/hellosign/sdk/HelloSignClientTest.java

```diff
@@ -71,6 +71,7 @@
  */
 
 public class HelloSignClientTest {
+    public static final String SIGNATURE_REQUEST_ID = "fa5c8a0b0f492d768749333ad6fcc214c111e967";
```

Comment:
I'd rather this just be hard-coded in the test (the value actually doesn't matter since we mock the request).